js/babel-plugin-sprockets-commoner-internal/index.js

```diff
@@ -171,14 +181,18 @@ module.exports = function (context) {
 
       var replacement = resolveTarget(state.file, target);
       switch (path.parent.type) {
-        case "ExpressionStatement":
-          // We just need to know there's a dependency, we can remove it then
-          path.remove();
-          break;
-        default:
-          // Otherwise we just look for the module by referencing its Special Identifier™
+      case "ExpressionStatement":
+        // We just need to know there's a dependency, we can remove it then
+        path.remove();
+        break;
+      default:
+        // Otherwise we just look for the module by referencing its Special Identifier™
+        if (replacement === false) {
+          path.replaceWith(t.objectExpression([]));
```

Comment:
what do you mean by empty function? something like 

`console.log(require('stubbed'))` would become `console.log({})`